src/util/Constants.js

```diff
@@ -81,12 +81,14 @@ exports.DefaultOptions = {
    * @property {string} [api='https://discord.com/api'] Base url of the API
    * @property {string} [cdn='https://cdn.discordapp.com'] Base url of the CDN
    * @property {string} [invite='https://discord.gg'] Base url of invites
+   * @property {string} [template='https://discord.com/template'] Base url of templates
    */
   http: {
     version: 7,
     api: 'https://discord.com/api',
     cdn: 'https://cdn.discordapp.com',
     invite: 'https://discord.gg',
+    template: 'https://discord.com/template',
```

Comment:
The client is using this other domain in the ui.
```suggestion
    template: 'https://discord.new',
```

It looks like neither endpoint / domain is documented however, or is this explicitly documented somewhere?

Reply:
discord.gg isn't documented anywhere either (they both should probably be added to the reference page)

Reply:
`https://discord.new/:template_code` seems to redirect to `https://discord.com/template/:template_code`, so I thought that maybe Discord was perhaps using the `.new` domain for templates temporarily or something (?) which is why I decided to go with `.com/template`. I could be wrong, but I feel like it would be a safer bet to use this one anyways?

Reply:
It's not temporary iirc, just like how the gift domain exists, there's one for templates